<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ComputerHousekeeper</class>
    <widget class="QMainWindow" name="ComputerHousekeeper">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>ComputerHousekeeper</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget#ComputerHousekeeper1_0
                {
                background-color:white
                }
            </string>
        </property>
        <property name="iconSize">
            <size>
                <width>32</width>
                <height>32</height>
            </size>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QFrame" name="frame_panel">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>64</width>
                        <height>618</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QFrame#frame_panel
                        {
                        background-color:white
                        }
                    </string>
                </property>
                <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                </property>
                <widget class="QPushButton" name="pushButton_task">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>0</y>
                            <width>64</width>
                            <height>64</height>
                        </rect>
                    </property>
                    <property name="contextMenuPolicy">
                        <enum>Qt::CustomContextMenu</enum>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">QPushButton#pushButton_task
                            {
                            border:0px solid rgb(16,124,16);
                            background-color:white
                            }
                            QPushButton#pushButton_task:hover
                            {
                            border:2px solid rgb(16,124,16);
                            border-color:green
                            }
                            QPushButton#pushButton_task:pressed
                            {
                            background-color:rgb(195, 195, 195)
                            }
                        </string>
                    </property>
                    <property name="text">
                        <string>任务</string>
                    </property>
                    <property name="iconSize">
                        <size>
                            <width>32</width>
                            <height>32</height>
                        </size>
                    </property>
                </widget>
                <widget class="QPushButton" name="pushButton_tool">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>64</y>
                            <width>64</width>
                            <height>64</height>
                        </rect>
                    </property>
                    <property name="contextMenuPolicy">
                        <enum>Qt::CustomContextMenu</enum>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">QPushButton#pushButton_tool
                            {
                            border:0px solid rgb(16,124,16);
                            background-color:white
                            }
                            QPushButton#pushButton_tool:hover
                            {
                            border:2px solid rgb(16,124,16);
                            border-color:green
                            }
                            QPushButton#pushButton_tool:pressed
                            {
                            background-color:rgb(195, 195, 195)
                            }
                        </string>
                    </property>
                    <property name="text">
                        <string>工具</string>
                    </property>
                    <property name="iconSize">
                        <size>
                            <width>32</width>
                            <height>32</height>
                        </size>
                    </property>
                </widget>
                <widget class="QPushButton" name="pushButton_more">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>128</y>
                            <width>64</width>
                            <height>64</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <weight>50</weight>
                            <bold>false</bold>
                            <strikeout>false</strikeout>
                            <stylestrategy>PreferDefault</stylestrategy>
                        </font>
                    </property>
                    <property name="contextMenuPolicy">
                        <enum>Qt::CustomContextMenu</enum>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">QPushButton#pushButton_more
                            {
                            border:0px solid rgb(16,124,16);
                            background-color:white
                            }
                            QPushButton#pushButton_more:hover
                            {
                            border:2px solid rgb(16,124,16);
                            border-color:green
                            }
                            QPushButton#pushButton_more:pressed
                            {
                            background-color:rgb(195, 195, 195)
                            }
                        </string>
                    </property>
                    <property name="text">
                        <string>更多</string>
                    </property>
                    <property name="iconSize">
                        <size>
                            <width>32</width>
                            <height>32</height>
                        </size>
                    </property>
                    <property name="checkable">
                        <bool>false</bool>
                    </property>
                    <property name="autoRepeat">
                        <bool>false</bool>
                    </property>
                    <property name="autoExclusive">
                        <bool>false</bool>
                    </property>
                    <property name="autoDefault">
                        <bool>false</bool>
                    </property>
                    <property name="default">
                        <bool>false</bool>
                    </property>
                    <property name="flat">
                        <bool>false</bool>
                    </property>
                </widget>
            </widget>
            <widget class="QFrame" name="frame_ui">
                <property name="geometry">
                    <rect>
                        <x>64</x>
                        <y>0</y>
                        <width>934</width>
                        <height>618</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">QFrame#frame_ui
                        {
                        background-color:white
                        }
                    </string>
                </property>
                <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
